site/docs/plugins/router.md

```diff
@@ -135,10 +137,10 @@ router.route("idle", async (ctx) => {
   await ctx.reply("Send /birthday to find out how long you have to wait.");
 });
 
-bot.use(router); // register the router
+bot.use(router); // Register the router.
 bot.start();
 
-// Date conversion utils:
+// Date conversion utils
```

Comment:
```suggestion
// Date conversion utils.
```

Reply:
We have said that we won't add a fullstop if the comment wasn't a sentence.

Reply:
We have said that we won't add a fullstop if the comment wasn't a sentence.